buffalo/cmd/setup.go

```diff
@@ -3,13 +3,13 @@ package cmd
 import (
 	"bytes"
 	"context"
-	"fmt"
 	"os"
 	"os/exec"
 	"strings"
 
 	"golang.org/x/sync/errgroup"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
The repo is now lower case `sirupsen`. Can you please undo these import changes?